reana_server/reana_admin/cli.py

```diff
@@ -639,3 +640,77 @@ def set_default_quota_limit(ctx):
             resource_type=resource.type_.name,
             limit=DEFAULT_QUOTA_LIMITS.get(resource.type_.name),
         )
+
+
+@reana_admin.command("consume-queue")
+@click.option(
+    "--queue-name",
+    "-q",
+    required=True,
+    type=str,
+    help="Name of the queue that will be consumed, e.g workflow-submission",
+)
+@click.option(
+    "--key",
+    "-k",
+    type=str,
+    help="Key of the property that will be used to filter the messages in the queue, e.g workflow_name_or_id",
+)
+@click.option(
+    "--values-to-delete",
+    "-v",
+    multiple=True,
+    help="List of property values to remove from the queue, e.g UUID of a workflow",
+)
+@click.option(
+    "-i",
+    "--interactive",
+    is_flag=True,
+    default=False,
+    help="Manually decide which messages to remove from the queue.",
+)
+def consume_queue(
+    queue_name: str,
+    key: Optional[str],
+    values_to_delete: List[str],
+    interactive: bool,
+):
+    """Start consuming specified queue and remove selected messages.
+
+    If -k option is specified, messages that have property values specified in -v will be deleted.
+
+    If -i option is specified, for every message, user will be asked what to do.
+
+    If -k and -i are specified together, for every message that matches property values in -v, user will be asked whateber to delete it or not.
+    """
+    from reana_server.reana_admin.consumer import MessageConsumer
+
+    if key is None and not interactive:
+        click.secho(
+            "Please provide -k (with -v) or -i options. These options can be used together or separately.",
+            fg="red",
+        )
+        sys.exit(1)
+
+    if key and not values_to_delete:
+        click.secho(
+            f"Please provide a list of message ids (using -m option) that need to be removed from the {queue_name} queue.",
+            fg="red",
+        )
+        sys.exit(1)
+
+    try:
+        consumer = MessageConsumer(
+            queue_name=queue_name,
+            key=key,
+            values_to_delete=list(values_to_delete),
+            is_interactive=interactive,
+        )
+    except Exception as error:
+        click.secho(
+            "Error is raised during MessageConsumer initialization. Please, check if arguments are correct.",
+            fg="red",
+        )
+        logging.exception(error)
```

Comment:
Do you think we need to show the stacktrace, or is the exception message enough?

Reply:
What do you mean by stacktrace? `logging.exception` is already printing stacktrace (and not a very helpful message 😄 )

Reply:
Sorry, I was not clear. I was wondering if it might be better not to show the stacktrace :smile:. What do you think about showing only the exception message instead of the full stacktrace?